src/main/scala/de/zalando/beard/filter/Filter.scala

```diff
@@ -19,6 +20,8 @@ case class WrongParameterTypeException(parameterName: String, paramterType: Stri
 case class TypeNotSupportedException(filterName: String, className: String) extends FilterException(filterName)
 case class FilterNotFound(filterName: String) extends FilterException(filterName)
 
+case class InputFormatException(filterName: String, message: String) extends FilterException(message)
```

Comment:
The filtername gets eaten up and is not shown in the console. It's better to prepend it to message.

```scala
case class IFE(filterName, message) extends FilterException(s"${filterName} - ${message}");
```